CMakeLists.txt

```diff
@@ -311,39 +308,46 @@ message(STATUS " ")
 
 # If it's an EMSCRIPTEN build, we're done
 if(EMSCRIPTEN)
-	return()
+  return()
 endif()
 
 
 # CMake exports
 configure_file(manifoldConfig.cmake.in ${CMAKE_CURRENT_BINARY_DIR}/manifoldConfig.cmake @ONLY)
 include(CMakePackageConfigHelpers)
 write_basic_package_version_file(
-	${CMAKE_BINARY_DIR}/cmake/manifoldConfigVersion.cmake
-	VERSION ${MANIFOLD_VERSION}
-	COMPATIBILITY SameMajorVersion
-	)
+  ${CMAKE_BINARY_DIR}/cmake/manifoldConfigVersion.cmake
+  VERSION ${MANIFOLD_VERSION}
+  COMPATIBILITY SameMajorVersion
+  )
 
 # Location of inputs for CMake find_package - see:
 # https://cmake.org/cmake/help/latest/command/find_package.html
 set(EXPORT_INSTALL_DIR ${CMAKE_INSTALL_LIBDIR}/cmake)
 
 install(EXPORT manifoldTargets
-	NAMESPACE manifold::
-  	DESTINATION ${EXPORT_INSTALL_DIR}/manifold
-	)
+  NAMESPACE manifold::
+  DESTINATION ${EXPORT_INSTALL_DIR}/manifold
+  )
 install(FILES
-	${CMAKE_CURRENT_BINARY_DIR}/cmake/manifoldConfigVersion.cmake
-	${CMAKE_CURRENT_BINARY_DIR}/manifoldConfig.cmake
-	DESTINATION ${EXPORT_INSTALL_DIR}/manifold
-	)
+  ${CMAKE_CURRENT_BINARY_DIR}/cmake/manifoldConfigVersion.cmake
+  ${CMAKE_CURRENT_BINARY_DIR}/manifoldConfig.cmake
+  DESTINATION ${EXPORT_INSTALL_DIR}/manifold
+  )
 
 
 # PkgConfig file
 if(MANIFOLD_CROSS_SECTION)
-	set(TEMPLATE_OPTIONAL_CLIPPER "Clipper2")
+  set(TEMPLATE_OPTIONAL_CLIPPER "Clipper2")
 endif()
 configure_file(manifold.pc.in ${CMAKE_CURRENT_BINARY_DIR}/manifold.pc @ONLY)
 install(FILES ${CMAKE_CURRENT_BINARY_DIR}/manifold.pc
-	DESTINATION ${CMAKE_INSTALL_LIBDIR}/pkgconfig)
+  DESTINATION ${CMAKE_INSTALL_LIBDIR}/pkgconfig)
+
 
+# Local Variables:
+# tab-width: 8
+# mode: cmake
+# indent-tabs-mode: t
+# End:
+# ex: shiftwidth=2 tabstop=8
```

Comment:
I can just format without adding the settings in the file if you'd rather not include something like that - I just wanted to offer that as an option to help folks keep the CMake formatting in line with Manifold's convention, since (at the moment) there isn't an automated tool enforcing it.

Reply:
Sure, that's fine by me - thanks for the explanation. I have almost never used vim or emacs, and sincerely hope I never need to.